src/main/java/hockeyapp/HockeyappApplication.java

```diff
@@ -46,7 +48,8 @@
     @DataBoundConstructor
     public HockeyappApplication(String apiToken, String appId, boolean notifyTeam,
                                 String filePath, String dsymPath, String libsPath,
-                                String tags, boolean downloadAllowed, OldVersionHolder oldVersionHolder,
+                                String tags, String teams, boolean mandatory, 
+                                boolean downloadAllowed, OldVersionHolder oldVersionHolder,
```

Comment:
Is that functionality in HockeyApp itself? Or order of fields here in Jenkins?
Reason i ask is cause lines 66/67 are in different order so viewing actual diffs was tricky.